workflow/envs/smoothcov.yaml

```diff
@@ -0,0 +1,9 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python>=3.6
+  - numpy>=1.0.0
+  - pip
+  - pip:
+    - argparse
```

Comment:
Python >=3.2 already includes argparse, so the three pip reqs lines can be removed.

Also, safer to request python >=3.7.